.github/workflows/release.yml

```
@@ -0,0 +1,93 @@
+name: Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      release-type:
+        type: choice
+        description: Release type
+        options:
+        - patch
+        - minor
+        - major
+        - prepatch
+        - preminor
+        - premajor
+        - prerelease
+        
+jobs:
+  test:
+    uses: "ambanum/OpenTermsArchive/.github/workflows/test.yml@main"
+  release:
+    needs: [ test ]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Git configuration
+        run: |
+          git config --global user.name "GitHub Actions"
+          git config --global user.email "github-action@users.noreply.github.com"
```

Why not use OTA bot for this?

---

I believe it is good to properly assign changes to the actor that is responsible for it. In this case, it is not @OTA-Bot but an unauthenticated GitHub Actions process that will create this commit 🙂 

However, let's unify name and email to the official GitHub Actions name:
```suggestion
          git config --global user.email "github-actions@users.noreply.github.com"
```